packages/hardware-trezor/test/transformers/txIn.test.ts

```diff
@@ -20,4 +20,21 @@ describe('tx-inputs', () => {
       });
     });
   });
+  describe('mapTxIns', () => {
+    it('can map a a set of TxIns', async () => {
+      const txIns = await mapTxIns([txIn, txIn, txIn], CONTEXT_WITH_KNOWN_ADDRESSES);
+
+      expect(txIns.length).toEqual(3);
+
+      for (const input of txIns) {
+        expect(input).toEqual({
```

Comment:
Just a thought, can we have one assertion for entire array instead of going through the txIns in a for-of loop? It could make the test a bit easier to read (it will show the exact structure the function will output)